### PR TITLE
Proof Request screen updates

### PR DESCRIPTION
**Pull request by amanji**
# Summary of Changes

Fixes Proof Request screen:
* Updates route params in screen
* Adds try/catch blocks to async operations
* Logs errors on failure and presents failure message

Replace this text with a high-level summary of the changes included in this PR.

N/A

Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [ ] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [ ] Run prettier: `npm run style-format`
- [ ] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: App/components/inputs/QRScanner.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ const QRScanner: React.FC<Props> = ({ handleCodeScan }) => {
           }}
         >
           <CameraViewContainer portrait={portraitMode}>
-            <CredentialOfferSteps activeStep={1} />
+            {/* <CredentialOfferSteps activeStep={1} /> */}
```
Comment (jleach) [inline App/components/inputs/QRScanner.tsx]: Remove dead code?

File: App/screens/ProofRequest.tsx [hunk 2]
```diff
@@ -2,42 +2,53 @@ import type { RouteProp } from '@react-navigation/native'
...
 
-import { SafeAreaScrollView, Button, ModularView, Label, Success, Pending, Failure } from 'components'
+import { Button, ModularView, Label, Success, Pending, Failure } from 'components'
 
 interface Props {
   navigation: StackNavigationProp<HomeStackParams, 'Proof Request'>
   route: RouteProp<HomeStackParams, 'Proof Request'>
 }
 
+const styles = StyleSheet.create({
```
Comment (jleach) [inline App/screens/ProofRequest.tsx:20]: Can we update to the new style system?

**Review by jleach — DISMISSED** [review]
A few trivial changes suggested but otherwise LGTM.

File: App/components/inputs/QRScanner.tsx (hunk 1 above)
Comment (amanji) [reply App/components/inputs/QRScanner.tsx]: Will do. Realized after the previous PR that the scanner is used for both Credential Offers and Proof Requests so the text added won't make sense. Will remove for now.

File: App/screens/ProofRequest.tsx (hunk 2 above)
Comment (amanji) [reply App/screens/ProofRequest.tsx:20]: Good catch. Will update.

Comment (amanji) [reply App/screens/ProofRequest.tsx:20]: Can I address these in a separate PR?

Comment (jleach) [reply App/screens/ProofRequest.tsx:20]: Sure.

Comment (amanji) [reply App/screens/ProofRequest.tsx:20]: Added issue #123 to track

**Review by jleach — APPROVED** [review]

**Pull request merged**